app/assets/javascripts/users.js

```diff
@@ -1,104 +1,45 @@
-onPage('users#edit, users#update', function() {
+onPage('users#edit, users#update', () => {
   // Hide/show avatar picker based on if the user is trying to to remove
   // the current avatar.
-  $('input#user_remove_avatar').on("change", function(e) {
-    var toDelete = e.currentTarget.checked;
+  $('input#user_remove_avatar').on('change', (e) => {
+    const toDelete = e.currentTarget.checked;
     $('.form-group.user_avatar').toggle(!toDelete);
-  }).trigger("change");
+  }).trigger('change');
 
-  var $approve_wca_id = $('#approve-wca-id');
-  var $unconfirmed_wca_id = $("#user_unconfirmed_wca_id");
-  var $unconfirmed_wca_id_profile_link = $("a#unconfirmed-wca-id-profile");
-  $approve_wca_id.on("click", function(e) {
-    $("#user_wca_id").val($unconfirmed_wca_id.val());
+  const $approve_wca_id = $('#approve-wca-id');
+  const $unconfirmed_wca_id = $('#user_unconfirmed_wca_id');
+  const $unconfirmed_wca_id_profile_link = $('a#unconfirmed-wca-id-profile');
+  $approve_wca_id.on('click', (e) => {
+    $('#user_wca_id').val($unconfirmed_wca_id.val());
     $unconfirmed_wca_id.val('');
     $unconfirmed_wca_id.trigger('input');
   });
-  $unconfirmed_wca_id.on("input", function(e) {
-    var unconfirmed_wca_id = $unconfirmed_wca_id.val();
-    $approve_wca_id.prop("disabled", !unconfirmed_wca_id);
+  $unconfirmed_wca_id.on('input', (e) => {
+    const unconfirmed_wca_id = $unconfirmed_wca_id.val();
+    $approve_wca_id.prop('disabled', !unconfirmed_wca_id);
     $unconfirmed_wca_id_profile_link.parent().toggle(!!unconfirmed_wca_id);
-    $unconfirmed_wca_id_profile_link.attr('href', "/persons/" + unconfirmed_wca_id);
+    $unconfirmed_wca_id_profile_link.attr('href', `/persons/${unconfirmed_wca_id}`);
   });
   $unconfirmed_wca_id.trigger('input');
 
   // Change the 'section' parameter when a tab is switched.
-  $('a[data-toggle="tab"]').on('show.bs.tab', function() {
-    var section = $(this).attr('href').slice(1);
-    window.wca.setUrlParams({ section: section });
+  $('a[data-toggle="tab"]').on('show.bs.tab', function () {
+    const section = $(this).attr('href').slice(1);
+    window.wca.setUrlParams({ section });
   });
   // Require the user to confirm reading guidelines.
-  $('#upload-avatar-form input[type="submit"]').on('click', function(event) {
-    var $confirmation = $('#guidelines-confirmation');
-    if(!$confirmation[0].checked) {
+  $('#upload-avatar-form input[type="submit"]').on('click', (event) => {
+    const $confirmation = $('#guidelines-confirmation');
+    if (!$confirmation[0].checked) {
       event.preventDefault();
       alert($confirmation.data('alert'));
     }
   });
 
   // Show avatar removal confirmation form
-  $(document).ready(function(){
-    $('.remove-avatar').click(function(){
+  $(document).ready(() => {
+    $('.remove-avatar').click(() => {
       $('.remove-avatar-confirm').show();
     });
   });
 });
```

Comment:
A bit nitpicky, but can you please revert this part of the file changes?
The only reason I'm asking is because I don't want the git blame to become "polluted" on this ancient part of the code base. It's important to be able to trace back changes and find out who wrote this and why, precisely _because_ it's ancient and arcane.

Reply:
I was hoping I will kill the rest of this file this week

Reply:
I will add all the files in assets/javascript to the eslint ignore so this doesn't happen again